### PR TITLE
Feature/getting started

### LINKED ISSUE #23: Write "getting started" documentation
Basic tutorial / intro for someone setting up a new project with parasol. Probably fine to make it django specific with notes about what would be different elsewhere.

This isn't something we need immediately, but would be nice to have and is important if/when we want to ask someone else to take parasol for a spin.

### PR DESCRIPTION
**Pull request by kmcelwee**
Closes #23

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -83,9 +83,8 @@ To use with Django:
 * Run ``solr_schema`` manage command to configure your schema; it will
   prompt to create the Solr core if it does not exist.
 
-.. Note::
-   The `SolrSchema` must be imported somewhere for it to be
-   found automatically.
+Note: The ``SolrSchema`` must be imported somewhere for it to be found
```
Comment (rlskoeser) [inline README.rst]: Why did you change this?

**Review by rlskoeser — COMMENTED** [review]
This is a great start! I have some thoughts about wording changes and generalizing a couple of things, but I'm not sure comments on a PR is a very good way to do that kind of editing and revision. I'm ok with merging it and then I can do some revisions after that, what do you think?

Why did you remove the note in the readme? This is one of the things github doesn't render very well but is called out nicely in the sphinx docs, my inclination is to keep it unless you can find a solution that works equally well for both.

**Comment by kmcelwee** [comment]
I assumed you'd want to reword things, no problem! And the Note deletion was carried over from earlier commits when I wasn't thinking about how the Readme was published elsewhere. I agree we should keep the Note tag!
